utils/sources.go

```diff
@@ -16,6 +16,12 @@ type ioClient struct {
 type IoSourceFactory struct {
 }
 
+func (c *ioClient) HealthStatus() v1.HealthStatus {
+	return v1.HealthStatus{
+		"": v1.Healthy,
```

Comment:
Why do we use a map for v1.HealthStatus if it's always an empty string key?

Reply:
It designed for composability, so we can combine multiple health check and a component can reply with checks of sub-components. I can add a utility const/function to make it cleaner for inits like that

Reply:
Sure, but maybe if in most cases we just want to return 1 value, we could use an interface for that with different implementing structs instead of empty string. But a utility also works